docs/talk/custom.scss

```
@@ -0,0 +1,30 @@
+/*-- scss:defaults --*/
+$font-family-sans-serif: "Inter", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)